requests/models.py

```diff
@@ -335,17 +337,11 @@ def prepare_method(self, method):
         self.method = method
         if self.method is None:
             raise ValueError('Request method cannot be "None"')
-        self.method = to_native_string(self.method).upper()
+        self.method = to_native_string(self.method.upper())
```

Comment:
Yep, this is purely stylistic I think. Just wanted to make sure we knew the change was being reverted.